app/assets/javascripts/govuk_publishing_components/lib/govspeak/youtube-link-enhancement.js

```diff
@@ -1,50 +1,101 @@
-// = require govuk_publishing_components/vendor/jquery-ui-1.10.2.custom
-// = require govuk_publishing_components/vendor/jquery.player.min
+window.GOVUK = window.GOVUK || {}
+// linting is not happy unless YT is defined
+var YT
```

Comment:
This looks a bit fishy here in the global scope. I think there are comments we can use to tell the linter to not complain about a variable.